packages/tokamak/contracts/hardhat.config.ts

```diff
@@ -20,6 +20,25 @@ import './tasks'
 // Load environment variables from .env
 dotenv.config()
 
+const { TASK_COMPILE_SOLIDITY_GET_SOLC_BUILD } = require("hardhat/builtin-tasks/task-names");
+const path = require("path");
+
+subtask(TASK_COMPILE_SOLIDITY_GET_SOLC_BUILD, async (args, hre, runSuper) => {
```

Comment:
Is it needed to compile contracts using the 0.5.17 Solidity compiler?

Reply:
As I checked, we need 0.5.17 for compiling WETH